Homework_6.py

```diff
@@ -0,0 +1,65 @@
+# Homework_6
+import os
+from Homework_5_v3 import *
+from Homework_4 import normalize_string
+
+
+class Publisher_file:  # class for choosing type of post
+    def input_post(self, post_text='', post_param='', ):  # function for input data
+        file = open('posts_to_add.txt', 'r')
+        lines = file.readlines()
+        file.close()
+        for i in range(0, len(lines) - 1):
+            lines[i] = normalize_string(lines[i])
+            if lines[i] == '\n':
+                i = i + 1
+                pass
+            elif lines[i] == 'News.\n':
```

Comment:
What if file has incorrect structure? First line != "News" or contains "News" and smth else? Maybe it is worth to handle these situations